### PR TITLE
Issue/29/create inference engine app

### PR DESCRIPTION
**Pull request by marekhering**

### CODE REVIEW

File: inference_engine/src/storage/storage.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return np.asarray(pil_image)
+
+    @staticmethod
+    def set_answer_key_json(exam_path: str, json_value: tp.Dict):
+        Storage.set_exam_answer_json(exam_path, f"{Config.exam_storage.answer_key_filename}.json", json_value)
+
+    @staticmethod
+    def set_exam_answer_json(exam_path: str, exam_name: str, json_value: tp.Dict):
+        exam_name = (f"{exam_name}.json" if not exam_name.endswith(".json") else exam_name)
+        full_path = Storage._create_full_path(exam_path, exam_name)
```
Comment (leckijakub) [inline inference_engine/src/storage/storage.py:35]: @marekhering answers should be saved in 'answers` directory

File: inference_engine/src/controller.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+from .storage import Storage
+from .preprocessing import Preprocessing
+from .model import AnswerModel, IndexModel
+from .dto import CheckExamsDTO, CheckExamDTO, GenerateExamKeyDTO
+
+
+class Controller:
+    @staticmethod
+    def check_exam(request: CheckExamDTO):
+        return Controller._mark_detection(request.exam_path, request.exam_name)
```
Comment (leckijakub) [inline inference_engine/src/controller.py:16]: @marekhering  I'm not exactly sure what PEP is specifying here but you are returning output of _mark_detection which does not return anything

File: inference_engine/src/storage/storage.py (hunk 1 above)
Comment (marekhering) [reply inference_engine/src/storage/storage.py]: Yes, it should. It's now saving answers on directory specified in config

File: inference_engine/src/controller.py (hunk 2 above)
Comment (marekhering) [reply inference_engine/src/controller.py]: PEP said that return is unnecessary here, it is already done

**Review by leckijakub — APPROVED** [review]
LGTM

**Pull request merged**